src/main/java/core/Game.java

```diff
@@ -580,7 +580,7 @@ public final AbstractAction oneAction() {
                 if (debug)
                     System.out.printf("Game: %2d Tick: %3d\t%s%n", gameState.getGameID(), getTick(), action.getString(gameState));
 
-                agentTime += (System.nanoTime() - s);
+                agentTime = (System.nanoTime() - s);
```

Comment:
Was this not supposed to be an average? So summed to then be divided by ticks?

Reply:
There was not a common policy as nextTime and copyTime recorded the results for this particular action, while agentTime was a running total. This meant that the three of them were (very) inconsistent in the metrics.

This now makes them comparable when they are reported by the ComputationTimes metric (which is per ACTION).